src/components/inputs/ForcedChoiceInput.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
This one doesn't seem to have a `<doc>` part. Though maybe the docs are not supposed to be complete just in this PR.

Reply:
Ah, I'll add documentation. Thanks :)